Gems/Atom/Asset/ImageProcessingAtom/Code/Source/BuilderSettings/PresetSettings.h

```diff
@@ -25,6 +25,13 @@ namespace ImageProcessingAtom
         AZ_TYPE_INFO(PresetSettings, "{4F4DEC5C-48DD-40FD-97B4-5FB6FC7242E9}");
         AZ_CLASS_ALLOCATOR(PresetSettings, AZ::SystemAllocator, 0);
 
+        //! Custom overrides for how to handle the output format
+        enum OutputTypeHandling
+        {
+            USE_SPECIFIED_OUTPUT_TYPE = 0,
+            USE_INPUT_FORMAT_AND_BIT_DEPTH
```

Comment:
I think the code guideline suggests we use CamelCase for all the names include enums.

Reply:
Ah that's right, I will update these